plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/Library.java

```diff
@@ -114,16 +105,22 @@ void setSiteUri(URI siteUri) {
     this.siteUri = siteUri;
   }
 
-  public synchronized List<LibraryFile> getLibraryFiles() {
-    return new ArrayList<>(libraryFiles);
+  /**
+   * Returns the complete list of all transitive dependencies for this library.
```

Comment:
Maybe note that this can result in network traffic and may be relatively expensive.

Reply:
done